GOVERNANCE.md

```diff
@@ -0,0 +1,74 @@
+# kyverno Governance
+
+This document defines governance policies for kyverno project.
+
+## Principles
+
+The kyverno project community adheres to the following principles:
```

Comment:
Please change `kyverno` to `Kyverno` everywhere, except when referring to the CLI binary.

Reply:
Okay